2018-06-27 11:58:10.206 30450 DEBUG oslo_concurrency.lockutils [req-b50ff8bc-ba78-4214-9a57-6abd7f38d0e8 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Acquired semaphore "dhcp-agent-network-lock-e9133ad7-02a1-43ed-82d3-d5cab89b35fd" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 11:58:10.207 30450 INFO neutron.agent.dhcp.agent [req-b50ff8bc-ba78-4214-9a57-6abd7f38d0e8 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T09:58:08Z, description=, device_id=eb1bdbba-8078-4041-9baa-9b157a380df2, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'55736366-4888-4eab-ae6d-9acaaf76e185', u'ip_address': u'11.201.1.8'}], id=e89df4bc-e642-43c4-bdea-aa3d344b84e8, mac_address=fa:16:3e:3c:38:4a, name=, network_id=e9133ad7-02a1-43ed-82d3-d5cab89b35fd, port_security_enabled=True, project_id=af8e1bd4cae142f2b706a1d7c6949185, revision_number=3, security_groups=[u'a8910fd3-6c5c-400a-9fe9-a757c58ebcea'], status=DOWN, tags=[], tenant_id=af8e1bd4cae142f2b706a1d7c6949185, updated_at=2018-06-27T09:58:09Z
2018-06-27 11:58:10.208 30450 DEBUG neutron.agent.dhcp.agent [req-b50ff8bc-ba78-4214-9a57-6abd7f38d0e8 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Calling driver for network: e9133ad7-02a1-43ed-82d3-d5cab89b35fd action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 11:58:10.210 30450 DEBUG neutron.agent.linux.dhcp [req-b50ff8bc-ba78-4214-9a57-6abd7f38d0e8 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Building host file: /var/lib/neutron/dhcp/e9133ad7-02a1-43ed-82d3-d5cab89b35fd/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 11:58:10.216 30450 DEBUG neutron.agent.linux.dhcp [req-b50ff8bc-ba78-4214-9a57-6abd7f38d0e8 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Done building host file /var/lib/neutron/dhcp/e9133ad7-02a1-43ed-82d3-d5cab89b35fd/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 11:58:10.222 30450 DEBUG neutron.agent.linux.utils [req-b50ff8bc-ba78-4214-9a57-6abd7f38d0e8 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2272'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 11:58:10.243 30450 DEBUG neutron.agent.linux.dhcp [req-b50ff8bc-ba78-4214-9a57-6abd7f38d0e8 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Reloading allocations for network: e9133ad7-02a1-43ed-82d3-d5cab89b35fd reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 11:58:10.245 30450 DEBUG neutron.agent.linux.utils [req-b50ff8bc-ba78-4214-9a57-6abd7f38d0e8 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-e9133ad7-02a1-43ed-82d3-d5cab89b35fd', 'ip', '-4', 'route', 'list', 'dev', 'tap74b5eccb-b7'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 11:58:10.262 30450 DEBUG neutron.agent.linux.utils [req-b50ff8bc-ba78-4214-9a57-6abd7f38d0e8 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-e9133ad7-02a1-43ed-82d3-d5cab89b35fd', 'ip', '-6', 'route', 'list', 'dev', 'tap74b5eccb-b7'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 11:58:10.273 30450 DEBUG neutron.agent.linux.utils [req-b50ff8bc-ba78-4214-9a57-6abd7f38d0e8 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Unable to access /var/lib/neutron/external/pids/e9133ad7-02a1-43ed-82d3-d5cab89b35fd.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 11:58:10.274 30450 DEBUG neutron.agent.linux.utils [req-b50ff8bc-ba78-4214-9a57-6abd7f38d0e8 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Unable to access /var/lib/neutron/external/pids/e9133ad7-02a1-43ed-82d3-d5cab89b35fd.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 11:58:10.275 30450 DEBUG neutron.agent.linux.external_process [req-b50ff8bc-ba78-4214-9a57-6abd7f38d0e8 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] No process started for e9133ad7-02a1-43ed-82d3-d5cab89b35fd disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 11:58:10.275 30450 DEBUG oslo_concurrency.lockutils [req-b50ff8bc-ba78-4214-9a57-6abd7f38d0e8 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Releasing semaphore "dhcp-agent-network-lock-e9133ad7-02a1-43ed-82d3-d5cab89b35fd" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 11:58:13.175 30450 DEBUG oslo_concurrency.lockutils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Acquired semaphore "dhcp-agent-network-lock-e9133ad7-02a1-43ed-82d3-d5cab89b35fd" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 11:58:13.176 30450 INFO neutron.agent.dhcp.agent [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T09:58:08Z, description=, device_id=eb1bdbba-8078-4041-9baa-9b157a380df2, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'55736366-4888-4eab-ae6d-9acaaf76e185', u'ip_address': u'11.201.1.8'}], id=e89df4bc-e642-43c4-bdea-aa3d344b84e8, mac_address=fa:16:3e:3c:38:4a, name=, network_id=e9133ad7-02a1-43ed-82d3-d5cab89b35fd, port_security_enabled=True, project_id=af8e1bd4cae142f2b706a1d7c6949185, revision_number=4, security_groups=[u'a8910fd3-6c5c-400a-9fe9-a757c58ebcea'], status=DOWN, tags=[], tenant_id=af8e1bd4cae142f2b706a1d7c6949185, updated_at=2018-06-27T09:58:12Z
2018-06-27 11:58:13.178 30450 DEBUG neutron.agent.dhcp.agent [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Calling driver for network: e9133ad7-02a1-43ed-82d3-d5cab89b35fd action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 11:58:13.180 30450 DEBUG neutron.agent.linux.dhcp [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Building host file: /var/lib/neutron/dhcp/e9133ad7-02a1-43ed-82d3-d5cab89b35fd/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 11:58:13.182 30450 DEBUG neutron.agent.linux.dhcp [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Done building host file /var/lib/neutron/dhcp/e9133ad7-02a1-43ed-82d3-d5cab89b35fd/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 11:58:13.186 30450 DEBUG neutron.agent.linux.utils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2272'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 11:58:13.197 30450 DEBUG neutron.agent.linux.dhcp [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Reloading allocations for network: e9133ad7-02a1-43ed-82d3-d5cab89b35fd reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 11:58:13.198 30450 DEBUG neutron.agent.linux.utils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-e9133ad7-02a1-43ed-82d3-d5cab89b35fd', 'ip', '-4', 'route', 'list', 'dev', 'tap74b5eccb-b7'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 11:58:13.215 30450 DEBUG neutron.agent.linux.utils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-e9133ad7-02a1-43ed-82d3-d5cab89b35fd', 'ip', '-6', 'route', 'list', 'dev', 'tap74b5eccb-b7'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 11:58:13.228 30450 DEBUG neutron.agent.linux.utils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Unable to access /var/lib/neutron/external/pids/e9133ad7-02a1-43ed-82d3-d5cab89b35fd.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 11:58:13.229 30450 DEBUG neutron.agent.linux.utils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Unable to access /var/lib/neutron/external/pids/e9133ad7-02a1-43ed-82d3-d5cab89b35fd.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 11:58:13.230 30450 DEBUG neutron.agent.linux.external_process [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] No process started for e9133ad7-02a1-43ed-82d3-d5cab89b35fd disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 11:58:13.231 30450 DEBUG oslo_concurrency.lockutils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Releasing semaphore "dhcp-agent-network-lock-e9133ad7-02a1-43ed-82d3-d5cab89b35fd" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 11:58:14.305 30450 DEBUG oslo_concurrency.lockutils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Acquired semaphore "dhcp-agent-network-lock-e9133ad7-02a1-43ed-82d3-d5cab89b35fd" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 11:58:14.306 30450 INFO neutron.agent.dhcp.agent [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-27T09:58:08Z, description=, device_id=eb1bdbba-8078-4041-9baa-9b157a380df2, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'55736366-4888-4eab-ae6d-9acaaf76e185', u'ip_address': u'11.201.1.8'}], id=e89df4bc-e642-43c4-bdea-aa3d344b84e8, mac_address=fa:16:3e:3c:38:4a, name=, network_id=e9133ad7-02a1-43ed-82d3-d5cab89b35fd, port_security_enabled=True, project_id=af8e1bd4cae142f2b706a1d7c6949185, revision_number=5, security_groups=[u'a8910fd3-6c5c-400a-9fe9-a757c58ebcea'], status=DOWN, tags=[], tenant_id=af8e1bd4cae142f2b706a1d7c6949185, updated_at=2018-06-27T09:58:13Z
2018-06-27 11:58:14.307 30450 DEBUG neutron.agent.dhcp.agent [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Calling driver for network: e9133ad7-02a1-43ed-82d3-d5cab89b35fd action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 11:58:14.309 30450 DEBUG neutron.agent.linux.dhcp [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Building host file: /var/lib/neutron/dhcp/e9133ad7-02a1-43ed-82d3-d5cab89b35fd/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 11:58:14.311 30450 DEBUG neutron.agent.linux.dhcp [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Done building host file /var/lib/neutron/dhcp/e9133ad7-02a1-43ed-82d3-d5cab89b35fd/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 11:58:14.316 30450 DEBUG neutron.agent.linux.utils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2272'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 11:58:14.329 30450 DEBUG neutron.agent.linux.dhcp [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Reloading allocations for network: e9133ad7-02a1-43ed-82d3-d5cab89b35fd reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 11:58:14.331 30450 DEBUG neutron.agent.linux.utils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-e9133ad7-02a1-43ed-82d3-d5cab89b35fd', 'ip', '-4', 'route', 'list', 'dev', 'tap74b5eccb-b7'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 11:58:14.350 30450 DEBUG neutron.agent.linux.utils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-e9133ad7-02a1-43ed-82d3-d5cab89b35fd', 'ip', '-6', 'route', 'list', 'dev', 'tap74b5eccb-b7'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 11:58:14.369 30450 DEBUG neutron.agent.linux.utils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Unable to access /var/lib/neutron/external/pids/e9133ad7-02a1-43ed-82d3-d5cab89b35fd.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 11:58:14.370 30450 DEBUG neutron.agent.linux.utils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Unable to access /var/lib/neutron/external/pids/e9133ad7-02a1-43ed-82d3-d5cab89b35fd.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 11:58:14.371 30450 DEBUG neutron.agent.linux.external_process [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] No process started for e9133ad7-02a1-43ed-82d3-d5cab89b35fd disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 11:58:14.373 30450 DEBUG oslo_concurrency.lockutils [req-c72c46db-3b93-4451-9361-097eb6224902 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Releasing semaphore "dhcp-agent-network-lock-e9133ad7-02a1-43ed-82d3-d5cab89b35fd" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 11:58:21.492 30450 DEBUG oslo_concurrency.lockutils [req-f52193a1-7bee-4a9f-b60e-eaee50c43f87 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Acquired semaphore "dhcp-agent-network-lock-e9133ad7-02a1-43ed-82d3-d5cab89b35fd" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 11:58:21.494 30450 DEBUG neutron.agent.dhcp.agent [req-f52193a1-7bee-4a9f-b60e-eaee50c43f87 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Calling driver for network: e9133ad7-02a1-43ed-82d3-d5cab89b35fd action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 11:58:21.497 30450 DEBUG neutron.agent.linux.utils [req-f52193a1-7bee-4a9f-b60e-eaee50c43f87 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-e9133ad7-02a1-43ed-82d3-d5cab89b35fd', 'dhcp_release', 'tap74b5eccb-b7', '11.201.1.8', 'fa:16:3e:3c:38:4a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 11:58:21.520 30450 DEBUG neutron.agent.linux.dhcp [req-f52193a1-7bee-4a9f-b60e-eaee50c43f87 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Building host file: /var/lib/neutron/dhcp/e9133ad7-02a1-43ed-82d3-d5cab89b35fd/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 11:58:21.523 30450 DEBUG neutron.agent.linux.dhcp [req-f52193a1-7bee-4a9f-b60e-eaee50c43f87 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Done building host file /var/lib/neutron/dhcp/e9133ad7-02a1-43ed-82d3-d5cab89b35fd/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 11:58:21.530 30450 DEBUG neutron.agent.linux.utils [req-f52193a1-7bee-4a9f-b60e-eaee50c43f87 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2272'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 11:58:21.542 30450 DEBUG neutron.agent.linux.dhcp [req-f52193a1-7bee-4a9f-b60e-eaee50c43f87 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Reloading allocations for network: e9133ad7-02a1-43ed-82d3-d5cab89b35fd reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 11:58:21.545 30450 DEBUG neutron.agent.linux.utils [req-f52193a1-7bee-4a9f-b60e-eaee50c43f87 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-e9133ad7-02a1-43ed-82d3-d5cab89b35fd', 'ip', '-4', 'route', 'list', 'dev', 'tap74b5eccb-b7'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 11:58:21.561 30450 DEBUG neutron.agent.linux.utils [req-f52193a1-7bee-4a9f-b60e-eaee50c43f87 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-e9133ad7-02a1-43ed-82d3-d5cab89b35fd', 'ip', '-6', 'route', 'list', 'dev', 'tap74b5eccb-b7'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 11:58:21.577 30450 DEBUG neutron.agent.linux.utils [req-f52193a1-7bee-4a9f-b60e-eaee50c43f87 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Unable to access /var/lib/neutron/external/pids/e9133ad7-02a1-43ed-82d3-d5cab89b35fd.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 11:58:21.579 30450 DEBUG neutron.agent.linux.utils [req-f52193a1-7bee-4a9f-b60e-eaee50c43f87 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Unable to access /var/lib/neutron/external/pids/e9133ad7-02a1-43ed-82d3-d5cab89b35fd.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 11:58:21.580 30450 DEBUG neutron.agent.linux.external_process [req-f52193a1-7bee-4a9f-b60e-eaee50c43f87 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] No process started for e9133ad7-02a1-43ed-82d3-d5cab89b35fd disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 11:58:21.580 30450 DEBUG oslo_concurrency.lockutils [req-f52193a1-7bee-4a9f-b60e-eaee50c43f87 08ee1da566734ee49f7e1dab31032096 af8e1bd4cae142f2b706a1d7c6949185 - - -] Releasing semaphore "dhcp-agent-network-lock-e9133ad7-02a1-43ed-82d3-d5cab89b35fd" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 11:58:32.427 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 11:58:32.437 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.010s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 11:59:32.427 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 11:59:32.436 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.008s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 12:00:27.056 30450 DEBUG oslo_concurrency.lockutils [req-ab380de4-c268-47fd-9056-52bdacaf38f9 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:00:27.494 30450 DEBUG neutron.agent.dhcp.agent [req-dc6b9e1e-d553-436d-9f42-655fb6b06d18 - - - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:00:27.496 30450 DEBUG neutron.agent.linux.utils [req-dc6b9e1e-d553-436d-9f42-655fb6b06d18 - - - - -] Unable to access /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:00:27.497 30450 DEBUG neutron.agent.linux.dhcp [req-dc6b9e1e-d553-436d-9f42-655fb6b06d18 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-7c4a3053-733b-416a-b416-671cc6686a9c on network 7c4a3053-733b-416a-b416-671cc6686a9c does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-06-27 12:00:27.497 30450 DEBUG neutron.agent.linux.dhcp [req-dc6b9e1e-d553-436d-9f42-655fb6b06d18 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-7c4a3053-733b-416a-b416-671cc6686a9c on network 7c4a3053-733b-416a-b416-671cc6686a9c does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-06-27 12:00:30.333 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.351 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.371 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.387 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.400 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.416 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', 'link', 'set', 'tap252f8fca-4e', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.438 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-o', 'link', 'show', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.453 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-27 12:00:30.471 30450 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap252f8fca-4e) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 12:00:30.473 30450 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-06-27 12:00:30.475 30450 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap252f8fca-4e) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 12:00:30.476 30450 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'252f8fca-4eb2-4179-b256-81d0a8c1c273', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:fe:4c:e2'})), record=tap252f8fca-4e) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 12:00:30.502 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap252f8fca-4e', 'address', 'fa:16:3e:fe:4c:e2'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.512 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.525 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap252f8fca-4e', 'netns', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.551 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', 'link', 'set', 'tap252f8fca-4e', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.561 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', 'link', 'set', 'tap252f8fca-4e', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.573 30450 DEBUG oslo_concurrency.lockutils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Acquired semaphore "iptables-qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:00:30.574 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.592 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.609 30450 DEBUG neutron.agent.linux.iptables_manager [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-06-27 12:00:30.610 30450 DEBUG oslo_concurrency.lockutils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Releasing semaphore "iptables-qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:00:30.611 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-o', 'link', 'show', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.622 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', 'addr', 'show', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.634 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'addr', 'add', '11.107.1.2/24', 'scope', 'global', 'dev', 'tap252f8fca-4e', 'brd', '11.107.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.644 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.656 30450 DEBUG neutron.agent.linux.dhcp [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Setting IPv4 gateway for dhcp netns on net 7c4a3053-733b-416a-b416-671cc6686a9c to 11.107.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1164
2018-06-27 12:00:30.657 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.107.1.1', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.668 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-6', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.680 30450 DEBUG neutron.agent.linux.dhcp [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Cleaning stale devices for network 7c4a3053-733b-416a-b416-671cc6686a9c _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-06-27 12:00:30.680 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.688 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.700 30450 DEBUG neutron.agent.linux.dhcp [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:623
2018-06-27 12:00:30.702 30450 DEBUG neutron.agent.linux.dhcp [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/leases with contents:
1530180030 fa:16:3e:fe:4c:e2 11.107.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:651
2018-06-27 12:00:30.702 30450 DEBUG neutron.agent.linux.dhcp [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Building host file: /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 12:00:30.703 30450 DEBUG neutron.agent.linux.dhcp [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Done building host file /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 12:00:30.705 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Unable to access /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:00:30.706 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host', '--addn-hosts=/var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap252f8fca-4e', '--dhcp-range=set:tag0,11.107.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:30.734 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:00:30.735 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:00:30.740 30450 DEBUG neutron.agent.linux.external_process [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:00:30.741 30450 DEBUG oslo_concurrency.lockutils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:00:30.743 30450 DEBUG oslo_concurrency.lockutils [req-ab380de4-c268-47fd-9056-52bdacaf38f9 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:00:31.800 30450 DEBUG neutron.agent.dhcp.agent [req-506e4cff-636b-409d-88ce-3d9ec00aebd2 - - - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:00:31.801 30450 DEBUG neutron.agent.linux.dhcp [req-506e4cff-636b-409d-88ce-3d9ec00aebd2 - - - - -] Building host file: /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 12:00:31.809 30450 DEBUG neutron.agent.linux.dhcp [req-506e4cff-636b-409d-88ce-3d9ec00aebd2 - - - - -] Done building host file /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 12:00:31.817 30450 DEBUG neutron.agent.linux.utils [req-506e4cff-636b-409d-88ce-3d9ec00aebd2 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '3844'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:31.831 30450 DEBUG neutron.agent.linux.dhcp [req-506e4cff-636b-409d-88ce-3d9ec00aebd2 - - - - -] Reloading allocations for network: 7c4a3053-733b-416a-b416-671cc6686a9c reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 12:00:31.833 30450 DEBUG neutron.agent.linux.utils [req-506e4cff-636b-409d-88ce-3d9ec00aebd2 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:31.849 30450 DEBUG neutron.agent.linux.utils [req-506e4cff-636b-409d-88ce-3d9ec00aebd2 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-6', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:31.864 30450 DEBUG neutron.agent.linux.utils [req-506e4cff-636b-409d-88ce-3d9ec00aebd2 - - - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:00:31.865 30450 DEBUG neutron.agent.linux.utils [req-506e4cff-636b-409d-88ce-3d9ec00aebd2 - - - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:00:31.866 30450 DEBUG neutron.agent.linux.external_process [req-506e4cff-636b-409d-88ce-3d9ec00aebd2 - - - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:00:31.868 30450 DEBUG oslo_concurrency.lockutils [req-506e4cff-636b-409d-88ce-3d9ec00aebd2 - - - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:00:31.869 30450 DEBUG oslo_concurrency.lockutils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:00:31.870 30450 DEBUG neutron.agent.dhcp.agent [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T10:00:27Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-7c4a3053-733b-416a-b416-671cc6686a9c, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'a8e36d4c-dfd6-4143-b2c7-f4da132f5290', u'ip_address': u'11.107.1.2'}], id=252f8fca-4eb2-4179-b256-81d0a8c1c273, mac_address=fa:16:3e:fe:4c:e2, name=, network_id=7c4a3053-733b-416a-b416-671cc6686a9c, port_security_enabled=False, project_id=3889f3933c4d448a818b1556a39530ec, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=3889f3933c4d448a818b1556a39530ec, updated_at=2018-06-27T10:00:28Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:409
2018-06-27 12:00:31.871 30450 DEBUG oslo_concurrency.lockutils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:00:31.873 30450 DEBUG oslo_concurrency.lockutils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:00:31.874 30450 INFO neutron.agent.dhcp.agent [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-27T10:00:27Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-7c4a3053-733b-416a-b416-671cc6686a9c, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'a8e36d4c-dfd6-4143-b2c7-f4da132f5290', u'ip_address': u'11.107.1.2'}], id=252f8fca-4eb2-4179-b256-81d0a8c1c273, mac_address=fa:16:3e:fe:4c:e2, name=, network_id=7c4a3053-733b-416a-b416-671cc6686a9c, port_security_enabled=False, project_id=3889f3933c4d448a818b1556a39530ec, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=3889f3933c4d448a818b1556a39530ec, updated_at=2018-06-27T10:00:29Z
2018-06-27 12:00:31.875 30450 DEBUG neutron.agent.dhcp.agent [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:00:31.876 30450 DEBUG neutron.agent.linux.dhcp [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Building host file: /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 12:00:31.878 30450 DEBUG neutron.agent.linux.dhcp [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Done building host file /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 12:00:31.882 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '3844'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:31.895 30450 DEBUG neutron.agent.linux.dhcp [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Reloading allocations for network: 7c4a3053-733b-416a-b416-671cc6686a9c reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 12:00:31.897 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:31.910 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-6', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:00:31.922 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:00:31.923 30450 DEBUG neutron.agent.linux.utils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:00:31.925 30450 DEBUG neutron.agent.linux.external_process [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:00:31.926 30450 DEBUG oslo_concurrency.lockutils [req-07812f95-ce61-41c6-a312-d17997e9cb74 - - - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:00:32.427 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 12:00:32.431 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 12:01:08.190 30450 DEBUG oslo_concurrency.lockutils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:01:08.192 30450 INFO neutron.agent.dhcp.agent [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T10:01:06Z, description=, device_id=97937ec8-52e1-4467-afc3-5a438f34a724, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'a8e36d4c-dfd6-4143-b2c7-f4da132f5290', u'ip_address': u'11.107.1.1'}], id=dffa2b9b-a383-4595-9eaf-77a60943a21c, mac_address=fa:16:3e:ca:66:13, name=, network_id=7c4a3053-733b-416a-b416-671cc6686a9c, port_security_enabled=False, project_id=3889f3933c4d448a818b1556a39530ec, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=3889f3933c4d448a818b1556a39530ec, updated_at=2018-06-27T10:01:07Z
2018-06-27 12:01:08.194 30450 DEBUG neutron.agent.dhcp.agent [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:01:08.196 30450 DEBUG neutron.agent.linux.dhcp [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Building host file: /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 12:01:08.200 30450 DEBUG neutron.agent.linux.dhcp [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Done building host file /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 12:01:08.209 30450 DEBUG neutron.agent.linux.utils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Running command (rootwrap daemon): ['kill', '-HUP', '3844'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:08.236 30450 DEBUG neutron.agent.linux.dhcp [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Reloading allocations for network: 7c4a3053-733b-416a-b416-671cc6686a9c reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 12:01:08.243 30450 DEBUG neutron.agent.linux.utils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:08.273 30450 DEBUG neutron.agent.linux.utils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-6', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:08.297 30450 DEBUG neutron.agent.linux.utils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:08.299 30450 DEBUG neutron.agent.linux.utils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:08.301 30450 DEBUG neutron.agent.linux.external_process [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:01:08.303 30450 DEBUG oslo_concurrency.lockutils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:01:09.763 30450 DEBUG oslo_concurrency.lockutils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:01:09.764 30450 INFO neutron.agent.dhcp.agent [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T10:01:06Z, description=, device_id=97937ec8-52e1-4467-afc3-5a438f34a724, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'a8e36d4c-dfd6-4143-b2c7-f4da132f5290', u'ip_address': u'11.107.1.1'}], id=dffa2b9b-a383-4595-9eaf-77a60943a21c, mac_address=fa:16:3e:ca:66:13, name=, network_id=7c4a3053-733b-416a-b416-671cc6686a9c, port_security_enabled=False, project_id=3889f3933c4d448a818b1556a39530ec, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=3889f3933c4d448a818b1556a39530ec, updated_at=2018-06-27T10:01:07Z
2018-06-27 12:01:09.765 30450 DEBUG neutron.agent.dhcp.agent [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:01:09.766 30450 DEBUG neutron.agent.linux.dhcp [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Building host file: /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 12:01:09.768 30450 DEBUG neutron.agent.linux.dhcp [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Done building host file /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 12:01:09.773 30450 DEBUG neutron.agent.linux.utils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Running command (rootwrap daemon): ['kill', '-HUP', '3844'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:09.785 30450 DEBUG neutron.agent.linux.dhcp [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Reloading allocations for network: 7c4a3053-733b-416a-b416-671cc6686a9c reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 12:01:09.786 30450 DEBUG neutron.agent.linux.utils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:09.810 30450 DEBUG neutron.agent.linux.utils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-6', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:09.831 30450 DEBUG neutron.agent.linux.utils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:09.832 30450 DEBUG neutron.agent.linux.utils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:09.833 30450 DEBUG neutron.agent.linux.external_process [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:01:09.834 30450 DEBUG oslo_concurrency.lockutils [req-c579f675-4aec-40d5-9a7f-617a21a96ca2 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:01:12.823 30450 DEBUG oslo_concurrency.lockutils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:01:12.824 30450 INFO neutron.agent.dhcp.agent [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T10:01:06Z, description=, device_id=97937ec8-52e1-4467-afc3-5a438f34a724, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'a8e36d4c-dfd6-4143-b2c7-f4da132f5290', u'ip_address': u'11.107.1.1'}], id=dffa2b9b-a383-4595-9eaf-77a60943a21c, mac_address=fa:16:3e:ca:66:13, name=, network_id=7c4a3053-733b-416a-b416-671cc6686a9c, port_security_enabled=False, project_id=3889f3933c4d448a818b1556a39530ec, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=3889f3933c4d448a818b1556a39530ec, updated_at=2018-06-27T10:01:08Z
2018-06-27 12:01:12.825 30450 DEBUG neutron.agent.dhcp.agent [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:01:12.826 30450 DEBUG neutron.agent.linux.dhcp [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Building host file: /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 12:01:12.828 30450 DEBUG neutron.agent.linux.dhcp [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Done building host file /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 12:01:12.832 30450 DEBUG neutron.agent.linux.utils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '3844'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:12.843 30450 DEBUG neutron.agent.linux.dhcp [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Reloading allocations for network: 7c4a3053-733b-416a-b416-671cc6686a9c reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 12:01:12.845 30450 DEBUG neutron.agent.linux.utils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:12.860 30450 DEBUG neutron.agent.linux.utils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-6', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:12.874 30450 DEBUG neutron.agent.linux.utils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:12.875 30450 DEBUG neutron.agent.linux.utils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:12.876 30450 DEBUG neutron.agent.linux.external_process [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:01:12.877 30450 DEBUG oslo_concurrency.lockutils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:01:13.487 30450 DEBUG oslo_concurrency.lockutils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:01:13.490 30450 INFO neutron.agent.dhcp.agent [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-27T10:01:06Z, description=, device_id=97937ec8-52e1-4467-afc3-5a438f34a724, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'a8e36d4c-dfd6-4143-b2c7-f4da132f5290', u'ip_address': u'11.107.1.1'}], id=dffa2b9b-a383-4595-9eaf-77a60943a21c, mac_address=fa:16:3e:ca:66:13, name=, network_id=7c4a3053-733b-416a-b416-671cc6686a9c, port_security_enabled=False, project_id=3889f3933c4d448a818b1556a39530ec, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=3889f3933c4d448a818b1556a39530ec, updated_at=2018-06-27T10:01:13Z
2018-06-27 12:01:13.491 30450 DEBUG neutron.agent.dhcp.agent [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:01:13.493 30450 DEBUG neutron.agent.linux.dhcp [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Building host file: /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 12:01:13.495 30450 DEBUG neutron.agent.linux.dhcp [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Done building host file /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 12:01:13.499 30450 DEBUG neutron.agent.linux.utils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '3844'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:13.512 30450 DEBUG neutron.agent.linux.dhcp [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Reloading allocations for network: 7c4a3053-733b-416a-b416-671cc6686a9c reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 12:01:13.513 30450 DEBUG neutron.agent.linux.utils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:13.528 30450 DEBUG neutron.agent.linux.utils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-6', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:13.539 30450 DEBUG neutron.agent.linux.utils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:13.541 30450 DEBUG neutron.agent.linux.utils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:13.542 30450 DEBUG neutron.agent.linux.external_process [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:01:13.543 30450 DEBUG oslo_concurrency.lockutils [req-7aa22c9a-86e9-4f6c-8934-214b05c8921f - - - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:01:22.097 30450 DEBUG oslo_concurrency.lockutils [req-80a1b794-2561-4a80-b437-7b83fe07ebf8 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Acquired semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:01:22.098 30450 DEBUG oslo_concurrency.lockutils [req-80a1b794-2561-4a80-b437-7b83fe07ebf8 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Releasing semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:01:25.880 30450 DEBUG oslo_concurrency.lockutils [req-0a55ea78-b0d3-4ab4-80a8-41a4e87e5654 - - - - -] Acquired semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:01:25.881 30450 DEBUG oslo_concurrency.lockutils [req-0a55ea78-b0d3-4ab4-80a8-41a4e87e5654 - - - - -] Releasing semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:01:26.634 30450 DEBUG oslo_concurrency.lockutils [req-0a55ea78-b0d3-4ab4-80a8-41a4e87e5654 - - - - -] Acquired semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:01:26.635 30450 DEBUG oslo_concurrency.lockutils [req-0a55ea78-b0d3-4ab4-80a8-41a4e87e5654 - - - - -] Releasing semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:01:32.428 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 12:01:32.438 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.010s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 12:01:46.203 30450 DEBUG oslo_concurrency.lockutils [req-2a0b2a6f-daab-4d7a-a6e2-f94d80949699 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:01:46.203 30450 INFO neutron.agent.dhcp.agent [req-2a0b2a6f-daab-4d7a-a6e2-f94d80949699 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T10:01:45Z, description=, device_id=cb5a88d8-b982-461a-9c95-eeeb2aa00b24, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'a8e36d4c-dfd6-4143-b2c7-f4da132f5290', u'ip_address': u'11.107.1.12'}], id=6871fff5-2c6e-4c48-b230-3d80c7360524, mac_address=fa:16:3e:af:ba:b9, name=, network_id=7c4a3053-733b-416a-b416-671cc6686a9c, port_security_enabled=True, project_id=3889f3933c4d448a818b1556a39530ec, revision_number=3, security_groups=[u'e45239a8-18a8-4a18-84fb-5b420351ba7d'], status=DOWN, tags=[], tenant_id=3889f3933c4d448a818b1556a39530ec, updated_at=2018-06-27T10:01:45Z
2018-06-27 12:01:46.205 30450 DEBUG neutron.agent.dhcp.agent [req-2a0b2a6f-daab-4d7a-a6e2-f94d80949699 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:01:46.206 30450 DEBUG neutron.agent.linux.dhcp [req-2a0b2a6f-daab-4d7a-a6e2-f94d80949699 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Building host file: /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 12:01:46.210 30450 DEBUG neutron.agent.linux.dhcp [req-2a0b2a6f-daab-4d7a-a6e2-f94d80949699 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Done building host file /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 12:01:46.220 30450 DEBUG neutron.agent.linux.utils [req-2a0b2a6f-daab-4d7a-a6e2-f94d80949699 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Running command (rootwrap daemon): ['kill', '-HUP', '3844'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:46.236 30450 DEBUG neutron.agent.linux.dhcp [req-2a0b2a6f-daab-4d7a-a6e2-f94d80949699 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Reloading allocations for network: 7c4a3053-733b-416a-b416-671cc6686a9c reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 12:01:46.238 30450 DEBUG neutron.agent.linux.utils [req-2a0b2a6f-daab-4d7a-a6e2-f94d80949699 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:46.249 30450 DEBUG neutron.agent.linux.utils [req-2a0b2a6f-daab-4d7a-a6e2-f94d80949699 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-6', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:46.261 30450 DEBUG neutron.agent.linux.utils [req-2a0b2a6f-daab-4d7a-a6e2-f94d80949699 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:46.263 30450 DEBUG neutron.agent.linux.utils [req-2a0b2a6f-daab-4d7a-a6e2-f94d80949699 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:46.263 30450 DEBUG neutron.agent.linux.external_process [req-2a0b2a6f-daab-4d7a-a6e2-f94d80949699 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:01:46.264 30450 DEBUG oslo_concurrency.lockutils [req-2a0b2a6f-daab-4d7a-a6e2-f94d80949699 e21979a39cfb4a64a424b57f1e5c03df 3889f3933c4d448a818b1556a39530ec - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:01:47.669 30450 DEBUG oslo_concurrency.lockutils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:01:47.670 30450 INFO neutron.agent.dhcp.agent [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-27T10:01:45Z, description=, device_id=cb5a88d8-b982-461a-9c95-eeeb2aa00b24, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'a8e36d4c-dfd6-4143-b2c7-f4da132f5290', u'ip_address': u'11.107.1.12'}], id=6871fff5-2c6e-4c48-b230-3d80c7360524, mac_address=fa:16:3e:af:ba:b9, name=, network_id=7c4a3053-733b-416a-b416-671cc6686a9c, port_security_enabled=True, project_id=3889f3933c4d448a818b1556a39530ec, revision_number=4, security_groups=[u'e45239a8-18a8-4a18-84fb-5b420351ba7d'], status=DOWN, tags=[], tenant_id=3889f3933c4d448a818b1556a39530ec, updated_at=2018-06-27T10:01:46Z
2018-06-27 12:01:47.671 30450 DEBUG neutron.agent.dhcp.agent [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:01:47.673 30450 DEBUG neutron.agent.linux.dhcp [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Building host file: /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 12:01:47.675 30450 DEBUG neutron.agent.linux.dhcp [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Done building host file /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 12:01:47.679 30450 DEBUG neutron.agent.linux.utils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '3844'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:47.691 30450 DEBUG neutron.agent.linux.dhcp [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Reloading allocations for network: 7c4a3053-733b-416a-b416-671cc6686a9c reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 12:01:47.695 30450 DEBUG neutron.agent.linux.utils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:47.707 30450 DEBUG neutron.agent.linux.utils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-6', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:47.721 30450 DEBUG neutron.agent.linux.utils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:47.722 30450 DEBUG neutron.agent.linux.utils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:47.723 30450 DEBUG neutron.agent.linux.external_process [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:01:47.723 30450 DEBUG oslo_concurrency.lockutils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:01:48.369 30450 DEBUG oslo_concurrency.lockutils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:01:48.370 30450 INFO neutron.agent.dhcp.agent [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-27T10:01:45Z, description=, device_id=cb5a88d8-b982-461a-9c95-eeeb2aa00b24, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'a8e36d4c-dfd6-4143-b2c7-f4da132f5290', u'ip_address': u'11.107.1.12'}], id=6871fff5-2c6e-4c48-b230-3d80c7360524, mac_address=fa:16:3e:af:ba:b9, name=, network_id=7c4a3053-733b-416a-b416-671cc6686a9c, port_security_enabled=True, project_id=3889f3933c4d448a818b1556a39530ec, revision_number=5, security_groups=[u'e45239a8-18a8-4a18-84fb-5b420351ba7d'], status=DOWN, tags=[], tenant_id=3889f3933c4d448a818b1556a39530ec, updated_at=2018-06-27T10:01:47Z
2018-06-27 12:01:48.371 30450 DEBUG neutron.agent.dhcp.agent [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:01:48.373 30450 DEBUG neutron.agent.linux.dhcp [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Building host file: /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 12:01:48.375 30450 DEBUG neutron.agent.linux.dhcp [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Done building host file /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 12:01:48.377 30450 DEBUG neutron.agent.linux.utils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '3844'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:48.391 30450 DEBUG neutron.agent.linux.dhcp [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Reloading allocations for network: 7c4a3053-733b-416a-b416-671cc6686a9c reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 12:01:48.393 30450 DEBUG neutron.agent.linux.utils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:48.406 30450 DEBUG neutron.agent.linux.utils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-6', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:01:48.417 30450 DEBUG neutron.agent.linux.utils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:48.418 30450 DEBUG neutron.agent.linux.utils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:01:48.418 30450 DEBUG neutron.agent.linux.external_process [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:01:48.419 30450 DEBUG oslo_concurrency.lockutils [req-e312e19e-bd16-4a69-84c8-251517c1a665 5890a01f91614c928c650e60460b0cc2 e80c8f8d9a994fdcb38348700180bb81 - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:02:14.169 30450 DEBUG oslo_concurrency.lockutils [req-56acf6db-95ee-4f55-b660-9ba1b05faf74 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:02:14.171 30450 DEBUG oslo_concurrency.lockutils [req-56acf6db-95ee-4f55-b660-9ba1b05faf74 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Releasing semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:02:15.951 30450 DEBUG oslo_concurrency.lockutils [req-56acf6db-95ee-4f55-b660-9ba1b05faf74 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:02:15.955 30450 DEBUG oslo_concurrency.lockutils [req-56acf6db-95ee-4f55-b660-9ba1b05faf74 aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Releasing semaphore "dhcp-agent-network-lock-c6b5bb99-dcc0-4f71-b6f8-268e791b3287" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:02:32.428 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 12:02:32.436 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.008s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 12:03:32.428 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 12:03:32.431 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 12:04:32.429 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 12:04:32.442 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.013s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 12:05:32.429 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 12:05:32.443 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.014s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 12:06:32.430 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 12:06:32.446 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.016s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 12:06:50.114 30450 DEBUG oslo_concurrency.lockutils [req-925b3d67-2edb-4f2c-9819-29645fd1be9a aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:06:50.116 30450 DEBUG neutron.agent.dhcp.agent [req-925b3d67-2edb-4f2c-9819-29645fd1be9a aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:06:50.119 30450 DEBUG neutron.agent.linux.utils [req-925b3d67-2edb-4f2c-9819-29645fd1be9a aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'dhcp_release', 'tap252f8fca-4e', '11.107.1.12', 'fa:16:3e:af:ba:b9'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:06:50.150 30450 DEBUG neutron.agent.linux.dhcp [req-925b3d67-2edb-4f2c-9819-29645fd1be9a aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Building host file: /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 12:06:50.158 30450 DEBUG neutron.agent.linux.dhcp [req-925b3d67-2edb-4f2c-9819-29645fd1be9a aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Done building host file /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 12:06:50.164 30450 DEBUG neutron.agent.linux.utils [req-925b3d67-2edb-4f2c-9819-29645fd1be9a aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '3844'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:06:50.182 30450 DEBUG neutron.agent.linux.dhcp [req-925b3d67-2edb-4f2c-9819-29645fd1be9a aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Reloading allocations for network: 7c4a3053-733b-416a-b416-671cc6686a9c reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 12:06:50.184 30450 DEBUG neutron.agent.linux.utils [req-925b3d67-2edb-4f2c-9819-29645fd1be9a aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:06:50.207 30450 DEBUG neutron.agent.linux.utils [req-925b3d67-2edb-4f2c-9819-29645fd1be9a aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-6', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:06:50.229 30450 DEBUG neutron.agent.linux.utils [req-925b3d67-2edb-4f2c-9819-29645fd1be9a aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:06:50.234 30450 DEBUG neutron.agent.linux.utils [req-925b3d67-2edb-4f2c-9819-29645fd1be9a aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:06:50.235 30450 DEBUG neutron.agent.linux.external_process [req-925b3d67-2edb-4f2c-9819-29645fd1be9a aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:06:50.236 30450 DEBUG oslo_concurrency.lockutils [req-925b3d67-2edb-4f2c-9819-29645fd1be9a aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:07:32.430 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 12:07:32.433 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 12:07:41.795 30450 DEBUG oslo_concurrency.lockutils [req-52b49f39-38db-4450-955e-6e94292d3beb aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:07:41.796 30450 DEBUG neutron.agent.dhcp.agent [req-52b49f39-38db-4450-955e-6e94292d3beb aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:07:41.798 30450 DEBUG neutron.agent.linux.utils [req-52b49f39-38db-4450-955e-6e94292d3beb aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'dhcp_release', 'tap252f8fca-4e', '11.107.1.1', 'fa:16:3e:ca:66:13'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:07:41.827 30450 DEBUG neutron.agent.linux.dhcp [req-52b49f39-38db-4450-955e-6e94292d3beb aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Building host file: /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 12:07:41.833 30450 DEBUG neutron.agent.linux.dhcp [req-52b49f39-38db-4450-955e-6e94292d3beb aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Done building host file /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 12:07:41.843 30450 DEBUG neutron.agent.linux.utils [req-52b49f39-38db-4450-955e-6e94292d3beb aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '3844'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:07:41.856 30450 DEBUG neutron.agent.linux.dhcp [req-52b49f39-38db-4450-955e-6e94292d3beb aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Reloading allocations for network: 7c4a3053-733b-416a-b416-671cc6686a9c reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 12:07:41.857 30450 DEBUG neutron.agent.linux.utils [req-52b49f39-38db-4450-955e-6e94292d3beb aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-4', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:07:41.870 30450 DEBUG neutron.agent.linux.utils [req-52b49f39-38db-4450-955e-6e94292d3beb aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c', 'ip', '-6', 'route', 'list', 'dev', 'tap252f8fca-4e'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:07:41.884 30450 DEBUG neutron.agent.linux.utils [req-52b49f39-38db-4450-955e-6e94292d3beb aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:07:41.885 30450 DEBUG neutron.agent.linux.utils [req-52b49f39-38db-4450-955e-6e94292d3beb aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:07:41.886 30450 DEBUG neutron.agent.linux.external_process [req-52b49f39-38db-4450-955e-6e94292d3beb aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:07:41.886 30450 DEBUG oslo_concurrency.lockutils [req-52b49f39-38db-4450-955e-6e94292d3beb aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:08:09.896 30450 DEBUG oslo_concurrency.lockutils [req-87b395c8-eeb8-49ea-9342-943b4e5227ef aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:08:09.897 30450 DEBUG neutron.agent.dhcp.agent [req-87b395c8-eeb8-49ea-9342-943b4e5227ef aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:08:09.900 30450 DEBUG neutron.agent.linux.utils [req-87b395c8-eeb8-49ea-9342-943b4e5227ef aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command (rootwrap daemon): ['kill', '-9', '3844'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:08:09.917 30450 DEBUG neutron.agent.linux.utils [req-87b395c8-eeb8-49ea-9342-943b4e5227ef aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-27 12:08:09.974 30450 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tap252f8fca-4e) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 12:08:10.076 30450 DEBUG neutron.agent.linux.utils [req-085a3ceb-f31f-4828-b263-62c821ad8cf4 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:08:10.094 30450 DEBUG neutron.agent.linux.utils [req-085a3ceb-f31f-4828-b263-62c821ad8cf4 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:08:10.113 30450 DEBUG oslo_concurrency.lockutils [req-085a3ceb-f31f-4828-b263-62c821ad8cf4 - - - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:08:18.776 30450 DEBUG oslo_concurrency.lockutils [req-9ae4d53c-026c-4723-9d6d-038dfdf4e88e aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:08:19.104 30450 DEBUG neutron.agent.linux.utils [req-da10aefc-bd25-4edb-b5fc-8b0dea5e1ca6 - - - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:08:19.106 30450 DEBUG neutron.agent.linux.utils [req-da10aefc-bd25-4edb-b5fc-8b0dea5e1ca6 - - - - -] Unable to access /var/lib/neutron/external/pids/7c4a3053-733b-416a-b416-671cc6686a9c.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:08:19.106 30450 DEBUG neutron.agent.linux.external_process [req-da10aefc-bd25-4edb-b5fc-8b0dea5e1ca6 - - - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:08:19.109 30450 DEBUG neutron.agent.dhcp.agent [req-da10aefc-bd25-4edb-b5fc-8b0dea5e1ca6 - - - - -] Calling driver for network: 7c4a3053-733b-416a-b416-671cc6686a9c action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 12:08:19.110 30450 DEBUG neutron.agent.linux.utils [req-da10aefc-bd25-4edb-b5fc-8b0dea5e1ca6 - - - - -] Unable to access /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:08:19.111 30450 DEBUG neutron.agent.linux.utils [req-da10aefc-bd25-4edb-b5fc-8b0dea5e1ca6 - - - - -] Unable to access /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 12:08:19.112 30450 DEBUG neutron.agent.linux.external_process [req-da10aefc-bd25-4edb-b5fc-8b0dea5e1ca6 - - - - -] No process started for 7c4a3053-733b-416a-b416-671cc6686a9c disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 12:08:19.113 30450 DEBUG neutron.agent.linux.dhcp [req-da10aefc-bd25-4edb-b5fc-8b0dea5e1ca6 - - - - -] Impossibile accedere a /var/lib/neutron/dhcp/7c4a3053-733b-416a-b416-671cc6686a9c/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:268
2018-06-27 12:08:19.114 30450 DEBUG neutron.agent.linux.dhcp [req-da10aefc-bd25-4edb-b5fc-8b0dea5e1ca6 - - - - -] No interface exists for network 7c4a3053-733b-416a-b416-671cc6686a9c destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1482
2018-06-27 12:08:19.147 30450 DEBUG neutron.agent.linux.utils [req-964d9916-7946-416e-b29a-f1287c6faa1a - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 12:08:19.161 30450 DEBUG neutron.agent.linux.dhcp [req-964d9916-7946-416e-b29a-f1287c6faa1a - - - - -] Namespace already deleted: qdhcp-7c4a3053-733b-416a-b416-671cc6686a9c _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:246
2018-06-27 12:08:19.162 30450 DEBUG oslo_concurrency.lockutils [req-964d9916-7946-416e-b29a-f1287c6faa1a - - - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:08:20.520 30450 DEBUG oslo_concurrency.lockutils [req-9ae4d53c-026c-4723-9d6d-038dfdf4e88e aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Acquired semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 12:08:20.521 30450 DEBUG oslo_concurrency.lockutils [req-9ae4d53c-026c-4723-9d6d-038dfdf4e88e aa9ab289b900453cbe540da36aa76872 1a35e3b0e18c4eff8ec68ce1f34b28b0 - - -] Releasing semaphore "dhcp-agent-network-lock-7c4a3053-733b-416a-b416-671cc6686a9c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 12:08:26.808 30450 DEBUG neutron.agent.dhcp.agent [req-da529c0f-a0f0-400a-b1f6-ccfa41b83fd0 - - - - -] resync (7c4a3053-733b-416a-b416-671cc6686a9c): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:255
2018-06-27 12:08:26.809 30450 INFO neutron.agent.dhcp.agent [req-da529c0f-a0f0-400a-b1f6-ccfa41b83fd0 - - - - -] Synchronizing state
2018-06-27 12:08:27.542 30450 INFO neutron.agent.dhcp.agent [req-62a9ac82-7227-4d3a-8524-bb120f1852ce - - - - -] All active networks have been fetched through RPC.
2018-06-27 12:08:27.544 30450 INFO neutron.agent.dhcp.agent [-] Starting network c6b5bb99-dcc0-4f71-b6f8-268e791b3287 dhcp configuration
2018-06-27 12:08:27.545 30450 INFO neutron.agent.dhcp.agent [-] Finished network c6b5bb99-dcc0-4f71-b6f8-268e791b3287 dhcp configuration
2018-06-27 12:08:27.546 30450 INFO neutron.agent.dhcp.agent [req-62a9ac82-7227-4d3a-8524-bb120f1852ce - - - - -] Synchronizing state complete
2018-06-27 12:08:32.430 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 12:08:32.433 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 12:09:32.430 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 12:09:32.434 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 12:10:32.430 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 12:10:32.434 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 12:11:32.431 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 12:11:32.433 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 12:12:32.431 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 12:12:32.436 30450 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
